rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/v1/SwaggerDefinitionConfig.java

```diff
@@ -45,7 +45,7 @@
         consumes = {APPLICATION_JSON},
         produces = {APPLICATION_JSON},
         schemes = {SwaggerDefinition.Scheme.HTTP, SwaggerDefinition.Scheme.HTTPS},
-        externalDocs = @ExternalDocs(value = "Wiki", url = "https://github.com/gchq/Gaffer/wiki")
+        externalDocs = @ExternalDocs(value = "Wiki", url = "https://gchq.github.io/gaffer-doc/")
```

Comment:
Please can you change the name of it to "Documentation" instead of "Wiki". Perhaps you could create a constant for the value so you don't have it hardcoded in 2 places.